CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Version 1.3.0
+------
+* Support for new `shopify config analytics` command to enable/disable anonymous usage reporting
+* Bug fixes and minor improvements to internal components
```

Comment:
Since we have some 'relevant' notes, I wonder if we can drop the note on minor improvements